torch/autograd/gradcheck.py

```diff
@@ -112,16 +109,17 @@ def get_stride(size):
 
 
 def _get_numerical_jacobian(fn, inputs, outputs=None, target=None, eps=1e-3) -> List[Tuple[torch.Tensor, ...]]:
-    """Computes the numerical jacobian for a given fn and inputs. Returns M * N jacobians
-    where M is the number of input tensors that require grad, and N is the number of output
-    float/complex tensors.
+    """Computes the numerical Jacobian of `fn` at the provided input and with
```

Comment:
Maybe it would be clearer to say, Jacobian of `fn(inputs)` wrt to `target`. If not specified, targets are the input.

Side question: Is this target argument actually used?

Reply:
Yes, used in common_nn